docs/platforms/android/session-replay/configuration.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Configuration
+sidebar_order: 4110
+description: "Learn about the general Session Replay configuration fields."
+og_image: /og-images/platforms-android-session-replay-configuration.png
+---
+
+## General Configuration
```

Comment:
Is there a case to separate out Replay sample rates from the rest of the config options, like what is done in web? https://docs.sentry.io/platforms/javascript/session-replay/configuration/